index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Donice Portfolio</title>
+
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet" />
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <link rel="stylesheet" href="./style.css" />
+</head>
+
+<body>
+  <header>
+    <img src="./images/SideImage.svg" id="nav-image" alt="" />
+    <nav>
+      <span class="material-symbols-outlined"> menu </span>
+    </nav>
+    
+  </header>
+
+  <section>
+    <div id="about-me">
+      <h1>Donice Ubaru</h1>
+      <p>
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Take a look at my work.
+        <br><br>
+        If you like what you see and
+        have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <button type="button">Start Collaboration</button>
```

Comment:
[OPTIONA] 
- [ ] Kindly make sure to always use the required font family for each text in your project, following the `Figma design` is a project requirement and helps in terms of practicing.

Reply:
@Ofarouq310 can i merge now?